flint/coadd/linmos.py

```diff
@@ -259,6 +267,19 @@ def generate_linmos_parameter_set(
     # attribute drops the parent directory (absolute directory).
     parent_dir = linmos_names.image_fits.parent
 
+    # Work out the appropriate rotation to provide linmos. This should
```

Comment:
This is the part that most worries me. I think it is correct. But, who knows. Maybe you. You pirate.